VideoWeb/VideoWeb.Common/Models/Participant.cs

```diff
@@ -1,10 +1,12 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace VideoWeb.Common.Models
 {
     public class Participant
     {
+        private readonly string[] witnessRoles = { "witness", "expert" };
```

Comment:
sorry to nitpick - could we add a line break between the member and the constructor?